artefacts/README.md

```diff
@@ -1 +1,16 @@
+[<< Back](https://cntt-n.github.io/CNTT/)
+# Handling of figures
 
+## Table of Contents
+
+* [1 Usage of figures](#1)
+
+<a name="1"></a>
+## 1 Usage of figures
+
+- Figures used in the document should be in Portable Network Graphic (PNG) format
+- Figures must be stored in the `figures` folder of the document where the figures are used (e.g.: `doc/ref_arch/figures/`)
+- File name of the figures shold contain the chapter number where the figure is used as a prefix
```

Comment:
Just the chapter number as in "4" or as in "Ch04"?
In case we decide to reorganise the Figures would it be better if we asked to include the WS and chapter as a prefix. What about the Figure legend? should it also contain the Figure Number?

<WS acronym (do not include "-")>><"_Ch"><2-digit chapter number><"_"><figure legend (spaces replaced by "_")><"_Figure_"><figure number with "." or "-" replaced by "_">

Example, "RM_Ch04_Cloud_Infra_Figure_4_2"; "RA2_Ch04_k8s_architecture_Figure_4_1"

Reply:
I would not use the figure number, as they change too often ;)

Reply:
Giving the pictures names based on content rather than on where they are located would be good.  It's easy enough to search the repo for a filename to find out which chapters an image is referenced in.